.github/workflows/CI.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           - { name: "cairo", features: "png,pdf,svg,ps,use_glib,v1_18,freetype,script,xcb,xlib,win32-surface", nightly: "--features 'png,pdf,svg,ps,use_glib,v1_18,freetype,script,xcb,xlib,win32-surface'", test_sys: true }
           - { name: "gdk-pixbuf", features: "v2_42", nightly: "--all-features", test_sys: true }
           - { name: "gio", features: "v2_74", nightly: "--all-features", test_sys: true }
-          - { name: "glib", features: "v2_74", nightly: "--all-features", test_sys: true }
+          - { name: "glib", features: "v2_74", nightly: "--features=v2_76,log,log_macros,compiletests", test_sys: true }
```

Comment:
Well for nightly it's obviously fine to enable that feature. The way how it was before seemed good to me :) Sorry

I only didn't merge this yet because there was some CI failure and I had no time to look into that yet